app/server/server.ts

```diff
@@ -1,7 +1,12 @@
+const config = require('@hmcts/properties-volume').addTo(require('config'));
+const setupSecrets = require('./services/setupSecrets');
+
+// Setup secrets before loading the app
+setupSecrets.setupKeyVaultSecrets();
```

Comment:
if you import the function this can be changed like so:
```suggestion
setupKeyVaultSecrets();
```